app/Svg/Generator.hs

```diff
@@ -272,9 +272,7 @@ edgeToSVG styled path =
              then
                  mappend
                      (A.strokeWidth "2px") $
-                     A.style (textValue $ T.concat ["fill:",
-                          pathFill path,
```

Comment:
@ethanliu24 you have the right instinct about not just hardcoding `fill: none` to fix the issue, and wanting to use the existing `pathFill` attribute of the `path` value.

To achieve this you can use a conditional expression (in Haskell, this is just `if ... then ... else ...`) with the condition that if the `pathFill` is blank then use `"none"`, else use the pathFill value.